app/serializers/api/v2/headings/commodity_serializer.rb

```diff
@@ -18,15 +18,10 @@ class CommoditySerializer
                    :description_plain,
                    :parent_sid,
                    :validity_start_date,
-                   :validity_end_date
+                   :validity_end_date,
+                   :declarable
```

Comment:
Added this to reflect the fact that we're now using fast declarable and this never got updated